src/hugolib/site_render.go

```diff
@@ -357,7 +357,13 @@ func (s *Site) renderAASA() error {
 		return errors.New("failed to create targetPath for Apple App Site Associate")
 	}
 
-	templ := s.lookupLayouts("apple_app_site_association.json", "_default/apple_app_site_association.json", "_internal/_default/apple_app_site_association.json")
+	var templ tpl.Template
+
+	if s.Cfg.GetInt("aasaVersion") == 1 {
+		templ = s.lookupLayouts("apple_app_site_association_v1.json", "_default/apple_app_site_association_v1.json", "_internal/_default/apple_app_site_association_v1.json")
```

Comment:
Might the file names be easier as `aasa_v1.json` and `aasa_v2.json`?

Reply:
agreed! file names have been updated.